priv/static/mods/pao_suukantsu.jq

```diff
@@ -0,0 +1,19 @@
+# Wait, what happens if both standard pao and suukantsu pao are in play, and Player A discards a hatsu into daisangen pao, but then player B discards a different tile into suukantsu pao? Which pao takes precedence?
+# Guess we find out the hard way.
+.after_call.actions += [
+  ["when", 
+    [
+      {"name": "status", "opts": ["daiminkan_pao_possible"]},
+      {"name": "match", "opts": [["last_call"], [[[["quad"], 1]]]]},
```

Comment:
same comment as for rinshan pao -- the quad check can be turned into a daiminkan check, eliminating the need for daiminkan_pao_possible